frontend/src/static/js/components/webstatus-overview-cells.ts

```diff
@@ -150,6 +151,18 @@ const renderFeatureName: CellRenderer = (feature, routerLocation, _options) => {
   return html` <a href=${featureUrl}>${feature.name}</a> `;
 };
 
+const renderChromiumUsage: CellRenderer = (
+  feature,
+  _routerLocation,
+  _options,
+) => {
+  if (feature.usage?.chromium?.daily && feature.usage.chromium.daily > 0) {
+    // Format to display percentage with single decimal e.g. 0.8371 -> 83.7%.
+    return html`${Math.round(feature.usage.chromium.daily * 1000) / 10}%`;
```

Comment:
To be consistent with how we do the rounding in [ChromeStatus](https://github.com/GoogleChrome/chromium-dashboard/blob/3a24ca2a904fd33a6eea3eff68ca673a440edfac/client-src/elements/chromedash-stack-rank-page.ts#L64)
```suggestion
    return html`${(feature.usage.chromium.daily * 100).toFixed(1)}%`;
```